src/Directory.Packages.props

```diff
@@ -32,6 +32,7 @@
     <PackageVersion Include="Npgsql" Version="7.0.1" />
     <PackageVersion Include="Polly" Version="7.2.3" />
     <PackageVersion Include="StyleCop.Analyzers" Version="1.1.118" />
+    <PackageVersion Include="System.CommandLine" Version="2.0.0-beta4.22272.1" />
```

Comment:
We should use a non-beta version. Is there anything that beta gives us specifically?